js/deer-config.js

```diff
@@ -36,6 +36,8 @@ export default {
         LOADED: "deer-loaded",
         NEW_VIEW: "deer-view",
         NEW_FORM: "deer-form",
+        VIEW_RENDERED : "deer-view-rendered",
```

Comment:
I'm okay with this for now.

Ultimately, this should be a prefix, so we only listen for the templates people are making and not every render. An example, "DEER-RENDERED-CAT" would only listen to the custom "CAT" template.